new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainInImageWdt.java

```diff
@@ -0,0 +1,238 @@
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.checkServiceExists;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createDockerRegistrySecret;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createSecretWithUsernamePassword;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.installAndVerifyOperator;
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+// Test to create model in image domain and verify the domain started successfully
+@DisplayName("Test to create domain in image domain using wdt and start the domain")
+@IntegrationTest
+class ItDomainInImageWdt implements LoggedTest {
+
+  private static HelmParams opHelmParams = null;
+  private static String opNamespace = null;
+  private static String operatorImage = null;
+  private static String domainNamespace = null;
+  private static ConditionFactory withStandardRetryPolicy = null;
+  private static String dockerConfigJson = "";
+  private String domainUid = "domain1";
+
+  private static Map<String, Object> secretNameMap;
+
+  /**
+   * Install Operator.
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    // create standard, reusable retry/backoff policy
+    withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+        .and().with().pollInterval(10, SECONDS)
+        .atMost(5, MINUTES).await();
+
+    // get a new unique opNamespace
+    logger.info("Creating unique namespace for Operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    logger.info("Creating unique namespace for Domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+  }
+
+  @Test
+  @DisplayName("Create domain in image domain")
+  @Slow
+  @MustNotRunInParallel
+  public void testCreateDomaininImageWdt() {
+    // admin/managed server name here should match with model yaml in WDT_MODEL_FILE
+    final String adminServerPodName = domainUid + "-admin-server";
+    final String managedServerPrefix = domainUid + "-managed-server";
+    final int replicaCount = 2;
+
+    // Create the repo secret to pull the image
+    createDockerRegistrySecret(domainNamespace);
+
+    // create secret for admin credentials
+    logger.info("Create secret for admin credentials");
+    String adminSecretName = "weblogic-credentials";
+    createSecretWithUsernamePassword(adminSecretName, domainNamespace, "weblogic", "welcome1");
```

Comment:
Use the following constant from TestConstants.java instead of hard-codinf weblogic/welcome1
public static final String ADMIN_USERNAME_DEFAULT = "weblogic";
public static final String ADMIN_PASSWORD_DEFAULT = "welcome1";

Reply:
Done